app/_src/kubernetes-ingress-controller/concepts/architecture.md

```diff
@@ -25,13 +25,13 @@ please see the [proxy](/gateway/latest/reference/proxy/) and [load balancing](/g
 
 In Kubernetes, there are several concepts that are used to logically identify workloads and route traffic between them.
 
-### Service / Pods
+### Service and pods
```

Comment:
https://kubernetes.io/docs/concepts/workloads/pods/. Hi @Guaris Pods are always capitalized in the K8s website. I know this because I was part of the group that made the P of Pods, D od Deployments, and S or services capital.

Reply:
Agree. If we refer to Kubernetes objects we should use the capitalised version for all of the `Pod`s, `Deployment`s and `Service`s.